server/src/routes/users.js

```diff
@@ -14,15 +14,16 @@ const testUsers = [
 ];
 
 // Insert test users if document is empty
-// eslint-disable-next-line func-names
-User.countDocuments(function(err, count) {
-  if (!err && count === 0) {
-    // eslint-disable-next-line func-names
-    User.insertMany(testUsers, function(err1) {
-      if (err1) throw err1;
-    });
-  }
-});
+insertTestUsers = () => {
+  User.countDocuments(function(err, count) {
```

Comment:
I was meaning this function. Eg
`User.countDocuments((err, count) => {`

Reply:
Done, Thanks for your help!